dev-lang/cobol/gnucobol-3.0~rc1.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A free COBOL compiler"
+DESCRIPTION="GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler."
+HOMEPAGE="https://sourceforge.net/projects/open-cobol/"
+SOURCE_URI="https://downloads.sf.net/open-cobol/gnu-cobol/gnucobol-3.0-rc1.tar.gz"
+REVISION="1"
+LICENSE="GNU GPL v3"
+COPYRIGHT="2001-2019 Free Software Foundation, Inc"
+CHECKSUM_SHA256="e55aeea6b1f77e763b4cd4b520c78eb06da7671b4dcc76463fd0237dbf2e4816"
+SOURCE_DIR="gnucobol-3.0-rc1"
+
+ARCHITECTURES="x86_64"
```

Comment:
Could you please test it on x86? You will have to add some $secondaryArchSuffix-es here and there.

Reply:
I don't have a x86 Haiku machine at the moment, went x86_64 on both my Haiku machines.
I could test it in a VM though.